packages/core/src/TimeAgo/TimeAgo.tsx

```diff
@@ -39,6 +39,8 @@ export type HvTimeAgoProps<C extends React.ElementType = "p"> =
       disableRefresh?: boolean;
       /** Whether to show seconds in the rendered time */
       showSeconds?: boolean;
+      /** Whether to show the time in 12-hour or 24-hour format */
+      twelveHour?: boolean;
```

Comment:
I think we shouldn't add this unless we have a specific & explicit request to do so.
The hour format is already inferred from the `locale` and in some places (OS-level), [users can configure it](https://help.d4h.com/article/400-changing-the-time-format) independently of the locale. I don't think we should facilitate "hyjacking" the format.

Initially, I didn't really have a strong opinion on this. But there's issues with the implementation (misaligned naming, 12h default, etc.), so I think we should not include it at all. Less code to test, less possible bugs

I'd like other opinions from the team though. What do you think @francisco-guilherme @MEsteves22 @HQFOX?